Use-cases/N4K-for-Image-Verification.md

```diff
@@ -0,0 +1,80 @@
+## Steps for image verification
+
+Below are the steps to verify images before deployment to Kubernetes runtime environments - 
+
+1. Download the certified N4K Kyverno and adapter images to the customer's private repo.
+2. Customize Kyverno and adapter deployment as required for the customer's environment via Helm values file (docker-registry credentials, custom CA, Proxy etc.).
+3. Deploy Kyverno using the Helm Chart.
+4. Deploy the adapters using the Helm Chart.
+5. Leverage cosign or Venafi workflow to sign the images.
+6. Deploy the image verification Kyverno policy.
+7. Confirm image verification based on policy pass/fail.
+
+
+
+## Location and Credentials to access N4K images
+
+Please download the Kyverno and adapter images below - 
+
+        ghcr.io/nirmata/kyverno:v1.8.1-n4kbuild.1
+        ghcr.io/nirmata/kyvernopre:v1.8.1-n4kbuild.1
+        ghcr.io/nirmata/kube-rbac-proxy:v0.13.1
+        ghcr.io/nirmata/nirmata-imagekey-controller:v0.1
```

Comment:
Do we really want to specify which exact tags to use in these instructions? When we upgrade, we'll have to come back and maintain this list.